index.html

```diff
@@ -1278,93 +1285,119 @@ <h2 id="transreq"><span data-profile="FPWD|CR|FPIG-NOTE|FPWG-NOTE|PR|REC|PR-OBSL
+<a href="https://lists.w3.org/Archives/Member/chairs/2010JanMar/0037.html">announcement to chairs</a>). </p>
+
+<p id="webreq-date">
+    Please send advance notice to webreq@w3.org:
+    <ul>
+        <li><strong>3 business days in advance</strong>: If you need help from the Webmaster to fix errors, or ask for Pubrules error exception.</li>
+        <li><strong>2 business days in advance</strong>: If the document is perfectly ready.</li>
+    </ul>
+</p>
 
-<li data-profile="CR" data-cr="editorial">
-Since there has been a previous Candidate Recommendation and the changes are <strong>only editorial</strong>, W3C can skip the Patent Policy Exclusion (see the <a href="https://www.w3.org/2003/12/22-pp-faq.html#nonewtext">Patent Policy FAQ</a>). If not, a proper <a href='transitions.html?profile=CR&amp;cr=substantive'>transition request</a> with the approval of the Director will be needed.</li>
+<h2 id="pubreq" id="webreq-template">Publication Request</h2>
 
-<li class="transreq-dir-approval" data-profile="FPWD|CR|PR|REC|OBSL|RSCND|FPIG-NOTE|FPWG-NOTE" data-cr="new|snapshot|rec-update|rec-amended" data-rec='new' data-notehistory="notchartered|first">Record of approval of the <a href="#transreq">transition request</a>.</li>
-<li class="transreq-dir-approval" data-profile="REC" data-rec='features|substantive'>Record of approval of the <a href="#updaterequest">update request</a>.</li>
-<li class="transreq-dir-approval" data-profile="FPIG-NOTE|FPWG-NOTE" data-notehistory="closure">Record of W3M decision to close the group.</li>
-<li class="transreq-dir-approval" data-profile="FPIG-NOTE|FPWG-NOTE" data-notehistory="none">Evidence that publication is in accordance with expectations set by the group
-charter (e.g., quote the charter).</li>
-</ol>
 
 <p class="domain-lead-aware" data-profile="CR" data-cr="new|snapshot|rec-update|rec-amended"><strong>Note:</strong>
-Someone from the W3C
-management team (usually the Project Management Lead) <span
-class="rfc2119">SHOULD</span> be aware of the status of the
-document.</p>
-
-<h2><a id="schedule-pubreq" name="schedule-pubreq">Scheduling
-Publication</a></h2>
+    Someone from the W3C
+    management team (usually the Project Management Lead) <span
+    class="rfc2119">SHOULD</span> be aware of the status of the
+    document.</p>
 
-<p class='note' data-profile="WD|CR|WG-NOTE|IG-NOTE" data-returning='false' data-cr="draft|snapshot" data-echidna='false'>
-  <strong>Tip:</strong> <span class="status-name">STATUS</span>s published through the <a href="https://github.com/w3c/echidna/wiki">W3C automatic system</a> do not need to get scheduled with the Webmaster and are not subjected to <a href="https://www.w3.org/Guide/#current">publishing moratoria</a>.
+<p> A publication request <span class="rfc2119">MUST</span> include the following information: <br>
+    <span class="gray-notes"> ---> You <span class="rfc2119">may</span> copy the list below and paste into the email sent to webreq@w3.org</span>
```

Comment:
nit: the hyphens are not pretty since they are not in the middle of the line vertically in many fonts. What about using characters like `→ ➞ ⟶` directly?